benchmarks/benchmark_utils.py

```diff
@@ -0,0 +1,14 @@
+"""Reocurring constants and methods for benchmarking functions."""
+
+import numpy as np
+
+
+RANDOM_5x5_SPD_MATRIX = np.array(
```

Comment:
Good idea to separate this out. We might even want to have a folder with matrices etc. similar as to the tests where we collect these benchmark / test problems. For now just a naming correction: `SPD_MATRIX_5x5` is more descriptive since it is not a random matrix.

Reply:
yes, I agree that putting those things into a separate folder is a good longterm goal. Regarding the naming: sounds sensible.